src/components/DataControllerMixin/DataControllerMixin.js

```diff
@@ -27,7 +28,32 @@ var ModalMixin = {
   },
 
   exportDataTable: function (format) {
-    exportTableDataToWorksheet(this.props, this.state.statsData, format, this.state.dataTableTimeOfYear);
+    exportDataToWorksheet("stats", this.props, this.state.statsData, format, this.state.dataTableTimeOfYear);
+  },
+
+  exportTimeSeries: function(format) {
+    var runs = _.map(this.props.meta, function(run) {
+      return this.getTimeseriesPromise(this.props, run.unique_id);
```

Comment:
Am I reading this correctly in that when the user clicks the download button, the UI will request the timeseries from the backend (again)? On one hand, this is somewhat redundant since the UI will have already downloaded this data to present it in the plot. On the other hand, the size is tiny, so re-downloading it isn't a huge deal. Just want to make sure that we're considering that. *Is* it easy to find this data in the existing state somewhere?